AK/Platform.h

```diff
@@ -27,6 +28,18 @@
 #    define AK_OS_BSD_GENERIC
 #endif
 
+#if defined(__CYGWIN__) || defined(_WIN32)
+#    define AK_OS_WIN32
+// FIXME: Do the following properly
+#    define O_CLOEXEC FD_CLOEXEC
```

Comment:
If you're going to do this, then you need to change like every single place we use O_CLOEXEC to fcntl(fd, FD_CLOEXEC) instead.... which seems horribly invasive. maybe just in Core::System::open?

Reply:
given the other O_CLOEXEC check in this PR, i'd say don't do this at all, and see what else breaks.